docs/api.md

```diff
@@ -891,6 +892,18 @@ stream:                 -2-4-5-6-8->
 stream.skipWhile(even): -----5-6-8->
 ```
 
+### skipAfter
+
+#### `stream.skipAfter(predicate) -> Stream`
+#### `most.skipAfter(predicate, stream) -> Stream`
+
+Create a new stream containing all events before and including when the `predicate` returns true.
+
+```
+stream:                 -1-2-4-5-6-8->
+stream.skipAfter(even): ---1-2->
```

Comment:
I don't think `skipAfter` will delay the events at all, and it should end immediately after the value where predicate becomes `true`.  Like this:

```
stream:                 -1-2-4-5-6-8->
stream.skipAfter(even): -1-2|
```